added screenshots for platform customization (#842)
SoftwareAG/c8y-docs: content/users-guide/enterprise-tenant-bundle/customization.md

Once the DNS configuration is in place and if a certificate with the given requirements is available, it can be easily
uploaded to the platform.

> In the **Domain name** tab under Enterprise Tenant, click **Upload certificate**. Then select the certificate from your file system and confirm upload by clicking on **Upload** button.

Reviewer: ```suggestion
On the **Domain name** tab in the **Enterprise Tenant** page, click **Upload certificate**. Select the certificate from your file system and click **Upload**.
```
